src/test/java/org/opensearch/security/dlic/rest/api/SavedTenantApiTest.java

```diff
@@ -0,0 +1,134 @@
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package org.opensearch.security.dlic.rest.api;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.opensearch.security.securityconf.impl.CType;
+import org.apache.http.Header;
+import org.apache.http.HttpStatus;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.xcontent.XContentType;
+import org.junit.Assert;
+import org.junit.Test;
+import org.opensearch.security.test.helper.rest.RestHelper.HttpResponse;
+import java.util.Arrays;
+
+import com.amazon.dlic.auth.http.saml.HTTPSamlAuthenticator;
+
+import static org.junit.Assert.*;
+
+@RunWith(Parameterized.class)
+public class SavedTenantApiTest extends AbstractRestApiUnitTest {
+    private final String BASE_ENDPOINT;
+    private final String ENDPOINT;
+
+    public SavedTenantApiTest(String baseEndpoint, String endpoint){
+        BASE_ENDPOINT = baseEndpoint;
+        ENDPOINT = endpoint;
+    }
+
+    @Parameterized.Parameters
+    public static Iterable<String[]> endpoints() {
+        return Arrays.asList(new String[][] {
+                {"/_opendistro/_security/api/", "/_opendistro/_security/api/account"},
+                {"/_plugins/_security/api/", "/_plugins/_security/api/account"}
+        });
+    }
+
+    @Test
+    public void testDefaultTenantValueCheck() throws Exception {
+        // arrange
+        setup();
+        final String testUser = "test-user";
+        final String testPass = "test-pass";
+        final String defaultTenantValue = "global-tenant";
+        final String savedTenantEndpoint = BASE_ENDPOINT + "account/saved_tenant";
+
+        // add user
+        addUserWithPassword(testUser, testPass, HttpStatus.SC_CREATED);
+
+        // test - unauthorized access as credentials are missing.
+        HttpResponse response = rh.executeGetRequest(savedTenantEndpoint);
+        assertEquals(HttpStatus.SC_UNAUTHORIZED, response.getStatusCode());
+
+        // test - incorrect password
+        response = rh.executeGetRequest(savedTenantEndpoint, encodeBasicHeader(testUser, testPass + "invalidating text"));
+        assertEquals(HttpStatus.SC_UNAUTHORIZED, response.getStatusCode());
+
+        // test - incorrect user; not possible to check until later (when adding a (target) user parameter in body)
+        // can't do this until user manager; check if only users who can manage this user can get/set this info
+
+        // test - valid request
+        response = rh.executeGetRequest(savedTenantEndpoint, encodeBasicHeader(testUser, testPass));
+        assertEquals(HttpStatus.SC_OK, response.getStatusCode());
+        Settings body = Settings.builder().loadFromSource(response.getBody(), XContentType.JSON).build();
+        assertEquals(defaultTenantValue, body.get("saved_tenant"));
+    }
+
+    @Test
+    public void testSet_tenant() throws Exception{
```

Comment:
Please use camel case with lower case first letter for method names. The method name `testDefaultTenantValueCheck` is good.